asv_bench/benchmarks/pandas/io/sas.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+from modin.pandas import read_sas
+
+
+class SAS:
+
+    params = ["sas7bdat", "xport"]
+    param_names = ["format"]
+
+    def setup(self, format):
+        # Read files that are located in 'pandas/tests/io/sas/data'
+        files = {"sas7bdat": "test1.sas7bdat", "xport": "paxraw_d_short.xpt"}
+        file = files[format]
+        paths = [
+            os.path.dirname(__file__),
+            "..",
+            "..",
+            "..",
+            "pandas",
+            "tests",
+            "io",
+            "sas",
+            "data",
+            file,
+        ]
+        self.f = os.path.join(*paths)
+
+    def time_read_sas(self, format):
+        read_sas(self.f, format=format)
```

Comment:
not sure this is important one

Reply:
agree, we are just defaulting to pandas for read_sas for now